### PR TITLE
adds a new clown mask, the madman, a madness combat reference

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request

adds a new clown mask and another "bit" reference 

## Why It's Good For The Game

1. we havent had a good pop culture reference in a while
2. MC4_Tricky_1

## Changelog
:cl: Thatguythere03, Jimmy Brickets
add: a new clown mask based of Madness Combat's Tricky! (Improbability drive not included)
/:cl:

### CODE REVIEW

File: code/modules/clothing/masks/gasmask.dm [hunk 1]
```diff
@@ -74,6 +74,7 @@
 	options["The Madman"] = "joker"
```
Comment (eeSPee) [inline code/modules/clothing/masks/gasmask.dm:74]: There's already a 'madman'

Comment (ToasterBan) [reply code/modules/clothing/masks/gasmask.dm:74]: oh right I'd didn't notice that I'll fix it in a bit

**Comment by EgorDinamit** [comment]
Add the road sign weapon as well.

**Comment by ToasterBan** [comment]
> Add the road sign weapon as well.

will actually add this as a clown Tator item once this gets merged ;)

**Review by utilbird — APPROVED** [review]

**Pull request merged**